src/db/firebaseDB/index.js

```diff
@@ -0,0 +1,41 @@
+const firebase = require('firebase/app')
+require('firebase/firestore')
+const config = require('./keys')
+const firebaseapp = firebase.initializeApp(config)
+
+const firestore = firebaseapp.firestore()
+const settings = {timestampsInSnapshots: true}
+firestore.settings(settings)
+
+const dbInfo = {
+  NOTES_COLLECTION: 'notes',
+  SESSION_ID_FIELD: 'sessionId'
+}
+
+module.exports = function () {
+  return firestore
+}
+
+function get (table, column, condition, data, callback) {
+  var query = firestore.collection(table)
+  if (data && condition && data) {
```

Comment:
The condition shouldn't be (column != null && condition != null && data != null)?